miner/bid_simulator.go

```diff
@@ -500,8 +506,13 @@ func (b *bidSimulator) simBid(interruptCh chan int32, bidRuntime *BidRuntime) {
 		blockNumber = bidRuntime.bid.BlockNumber
 		parentHash  = bidRuntime.bid.ParentHash
 		builder     = bidRuntime.bid.Builder
-		err         error
-		success     bool
+
+		bidTxs   = bidRuntime.bid.Txs
+		bidLen   = len(bidTxs)
+		payBidTx = bidTxs[bidLen-1]
```

Comment:
I am a bit confused here, the API level allow the no payBidTx, however here it assume that it must have the payBidTx.  We must maintain consistency.

Reply:
the API level does not allow no payBidTx for now